test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/Internal/TagHelperChunkDecoratorTest.cs

```diff
@@ -1,68 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Linq;
-using Microsoft.AspNetCore.Razor.Chunks;
-using Microsoft.AspNetCore.Razor.Parser.SyntaxTree;
-using Moq;
-using Xunit;
-
-namespace Microsoft.AspNetCore.Mvc.Razor.Host.Test
-{
-    public class TagHelperChunkDecoratorTest
-    {
-        #if OLD_RAZOR
-        [Fact]
-        public void Accept_CorrectlyDecoratesViewComponentChunks()
```

Comment:
This was covered in `ViewComponentTagHelperPassTest`. Just added another case to it,